pages/room/[...slug].js

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useRef, useState } from "react";
+import { useRouter } from 'next/router'
+import io from "socket.io-client";
+import Peer from "simple-peer";
+import Chat from '../../components/Spaces/Chat'
+import { Grid } from '@material-ui/core'
```

Comment:
Can you format your imports like the rest of the project? All external packages in one group. All local packages in another group. :D